packages/api-db-mongodb/src/db/comment.ts

```diff
@@ -161,23 +160,30 @@ export class MongoDBCommentAdapter implements DBCommentAdapter {
     }
   }
 
-  async getCommentsForItemByID(id: readonly string[]): Promise<PublicComment[]> {
+  async getPublicCommentsForItemByID(id: string): Promise<PublicComment[]> {
     const [comments] = await Promise.all([
       // TODO: add count
       // TODO: add sort revisions' array by createdAt
       this.comments
-        .aggregate([{$sort: {'createdAt.date': -1}}], {
-          collation: {locale: this.locale, strength: 2}
-        })
+        .aggregate(
+          [
+            {$sort: {'createdAt.date': -1}},
+            {$lookup: {from: 'users', localField: 'userID', foreignField: '_id', as: 'userID'}}
```

Comment:
Why do you get the users here?

Reply:
to populate the comment' author(user) details.

Reply:
fixed, now we are populating the user data in the field resolver function instead of the DB adapter